src/ccd2rgb.jl

```diff
@@ -0,0 +1,40 @@
+"""
+    ccd2rgb(red::ImageHDU, green::ImageHDU, blue::ImageHDU; stretch::Function = x -> x, shape_out = size(red))
+
+Converts 3 grayscale ImageHDU into RGB by reprojecting them.
+
+# Arguments
+- `red`: Red channel data.
+- `green`: Green channel data.
+- `blue`: Blue channel data.
+- `stretch`: Stretch function applied.
+- `shape_out`: Shape of output RGB image.
+
+# Examples
+```julia-repl
+julia> ccd2rgb(r, b, g, shape_out = (1000,1000))
+
+julia> ccd2rgb(r, b, g, shape_out = (1000,1000), stretch = x -> log(x))
```

Comment:
There is no need to use anonymous functions, just use the function name.  The same below and in the test file